app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -124,6 +133,11 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
 
     //Had to make this class variable, to implement various onClicks, which access the media, also I fell why make separate variables when one can serve the purpose
     private Media media;
+    private String reasonList[] = {"I uploaded it by mistake",
+            "I did not know it would be publicly visible",
+            "I realized it is bad for my privacy",
+            "Sorry this picture is not interesting for an encyclopedia",
+            "I changed my mind, I don't want it to be publicly visible anymore"};
```

Comment:
Sorry for the late notice, but how about an "Other (please detail on wiki)" reason too, for people who have a legitimate reason we have not thought about? :-)

Reply:
Also, I guess these strings should be extracted to `strings.xml`

Reply:
Definitely should be in strings.xml :) 

I am not sure if there is much difference between  "I did not know it would be publicly visible", , "I realized it is bad for my privacy", and "I changed my mind, I don't want it to be publicly visible anymore"? Would it be better to have just one option instead of those 3, that just says "Privacy concerns"?

Reply:
It would be great if we could finalize the list of reasons to be shown. The rest of the PR looks good to me and the PR can be merged once the strings have been finalized. 

@misaochan @nicolas-raoul

Reply:
@tanvidadu Are you around to make the changes ie putting the above strings to `strings.xml`?

Reply:
I will make the required changes !

Reply:
After rebasing the pr with current master, I am getting `Failed to find byte code for com/mapzen/android/lost/api/LostApiClient$ConnectionCallbacks` error. Any idea on how to resolve it without changing mapbox dependencies ?

Reply:
@tanvidadu Does it happen even on a clean build? 

Also, can you update the PR after rebasing so that we can try reproducing the issue?

Reply:
Yes @maskaravivek ,This error persists even on a clean build
I have updated the Pull request !

Reply:
I pulled your branch and tried building and running `prodDebug` and `betaDebug`. The above error didn't occur for me. Can you share the full stack trace? 

Eventually, if this error keeps happening for you then we can help in testing and get it merged. :)